### PR TITLE
Add blog post about Composite Transport.

### PR DESCRIPTION
**Pull request by JDarDagran**
blog post about Composite Transport in Java & Python.

### Checklist

- [ ] You've [signed-off](https://github.com/OpenLineage/OpenLineage/blob/main/why-the-dco.md) your work
- [ ] Your pull request title follows our [guidelines](https://github.com/OpenLineage/OpenLineage/blob/main/CONTRIBUTING.md#creating-pull-requests)
- [ ] Your changes are accompanied by tests (_if relevant_)
- [ ] Your change contains a [small diff](https://kurtisnusbaum.medium.com/stacked-diffs-keeping-phabricator-diffs-small-d9964f4dcfa6) and is self-contained
- [ ] You've updated any relevant documentation (_if relevant_)
- [ ] Your comment includes a one-liner for the changelog about the specific purpose of the change (_not required for changes to tests, docs, or CI config_)
- [ ] You've versioned the core OpenLineage model or facets according to [SchemaVer](https://docs.snowplowanalytics.com/docs/pipeline-components-and-applications/iglu/common-architecture/schemaver) (_if relevant_)
- [ ] You've added a [header](https://github.com/OpenLineage/OpenLineage/tree/main/.github/header_templates.md) to source files (_if relevant_)

----
SPDX-License-Identifier: Apache-2.0\
Copyright 2018-2024 contributors to the OpenLineage project

### CODE REVIEW

File: website/blog/composite-transport/index.mdx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      type: console
+```
+</TabItem>
+
+</Tabs>
+
+# Conclusion
+
+The Composite Transport is a powerful feature in OpenLineage that allows you to send events to multiple destinations sequentially.
+With its flexibility, convenience, and redundancy, it's an essential tool for anyone working with data lineage.
```
Comment (pawel-big-lebowski) [inline website/blog/composite-transport/index.mdx:113]: `With its flexibility, convenience, and redundancy, it's an essential tool for anyone working with data lineage.` -> What information is this sentence containing?

File: website/blog/composite-transport/index.mdx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    debug:
+      type: console
+```
+</TabItem>
+
+</Tabs>
+
+# Conclusion
+
+The Composite Transport is a powerful feature in OpenLineage that allows you to send events to multiple destinations sequentially.
```
Comment (arturowczarek) [inline website/blog/composite-transport/index.mdx:112]: This sentence is almost duplicate of the sentence at the beginning of the post.

Comment (JDarDagran) [reply website/blog/composite-transport/index.mdx:112]: I reworded it.

File: website/blog/composite-transport/index.mdx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    debug:
+      type: console
+```
+</TabItem>
+
+</Tabs>
+
+# Conclusion
+
+The Composite Transport enhances event distribution in OpenLineage by providing flexibility, redundancy,and ease of configuration.
```
Comment (arturowczarek) [inline website/blog/composite-transport/index.mdx:112]: missing space before `and`

**Review by arturowczarek — APPROVED** [review]
Just one missing space. Apart from that - LGTM.

File: website/blog/composite-transport/index.mdx (hunk 3 above)
Comment (JDarDagran) [reply website/blog/composite-transport/index.mdx]: Fixed.

**Review by pawel-big-lebowski — APPROVED** [review]

**Comment by pawel-big-lebowski** [comment]
@JDarDagran, you're the best!

**Pull request merged**